Fix GHA

h-vetinari opened the pull request:
Continue from #11

Closes #13

vicentebolea committed:
ci: correct conda settings

h-vetinari committed:
restrict github_actions_label to linux

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

h-vetinari committed:
remove fake name variable

h-vetinari committed:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.12.10.16.41.21

Other tools:
- conda-build 25.11.1
- rattler-build 0.53.0
- rattler-build-conda-compat 1.4.10

h-vetinari commented:
@vicentebolea, this works now.

vicentebolea commented:
Many thanks @h-vetinari
